pkgs/tools/security/tessen/default.nix

```
@@ -0,0 +1,42 @@
+{ lib
+, stdenvNoCC
+, fetchFromSourcehut
+, pass
+, bemenu
+, passBackend ? pass
+, picker ? bemenu
```

```suggestion
, pass
, bemenu
```

This is brittle. If a package named picker gets added this breaks. Also I don't see the reason why we would need this because the packages only get propagated to the final environment.

---

The tool supports different pickers and packends, so bemenu and pass are only one option out of a couple others. Is there a better way? or should I only just package the default ones and not provide a way to `override` them with other alternatives?

---

> or should I only just package the default ones and not provide a way to `override` them with other alternatives?

yes, we are only propagating them and doing nothing else with them which makes the override a bit overkill here.

---

Alright. How otherwise would I allow using any different pass backend and picker, though?